Generators/src/GeneratorsLinkDef.h

```diff
@@ -48,6 +48,7 @@
 #endif
 #pragma link C++ class o2::eventgen::GeneratorFromFile + ;
 #pragma link C++ class o2::eventgen::GeneratorFromO2Kine + ;
+#pragma link C++ class o2::conf::ConfigurableParamHelper < o2::eventgen::GeneratorFromO2KineParam> + ;
```

Comment:
I guess you need:
``` 
#pragma link C++ class o2::eventgen::GeneratorFromO2KineParam + ;
#pragma link C++ class o2::conf::ConfigurableParamHelper < o2::eventgen::GeneratorFromO2KineParam> + ;
```

This may solve the problem with the paramater that you described above.

Reply:
aha, thanks